<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="edit_start">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lab_start">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>起始状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_target">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>目标状态</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_target">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_begin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 110, 0);
selection-background-color: rgb(252, 233, 79);</string>
    </property>
    <property name="text">
     <string>begin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tab_Info">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>算法</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>算法</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>能否到达</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>open表</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>closed表</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>下拉选择</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BFS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DFS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>有序搜索</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lab_algo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>选择算法</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_depth">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>144</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lab_depth">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>146</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>深度限制</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumeau">
    <property name="title">
     <string>meau</string>
    </property>
    <addaction name="act_help"/>
   </widget>
   <addaction name="menumeau"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act_help">
   <property name="text">
    <string>help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>tab_Info</receiver>
   <slot>clearContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
